nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -505,9 +701,198 @@ def _impl(inputs, attr, params):
     'DepthwiseConv2dNative'             : _depthwise_conv(),
     'Shape'                             : _shape(),
     'Sigmoid'                           : AttrCvt('sigmoid'),
+    'Fill'                              : _fill(),
+    'GatherV2'                          : _gather_v2(),
+    'StridedSlice'                      : _stridedSlice(),
+}
+
+# _convert_map_rnn defines maps of rnn operator name to
+# converter functor(callable) for 1 to 1 mapping.
+_convert_map_rnn = {
+    'LSTMBlockCell'                     : _LSTMBlockCell(),
 }
 
 
+class RecurrentNetworks(object):
+    """Recurrent network layer handlers.
+
+    Unlike normal operators, stacked rnn have cells and layer concepts.
+    Each Layer represent a cell in RNN stack. Cells in the same RNN stack
+    sequentialy process input data.
```

Comment:
The description here is still a little confusing. Where is the recurrent part?
Can remove it or give a link to some RNN tutorial.